infrastructure/main.tf

```diff
@@ -60,5 +60,8 @@ module "send-letter-producer-service" {
     LETTER_TRACKING_DB_USER_NAME  = "${module.db.user_name}"
     LETTER_TRACKING_DB_PASSWORD   = "${module.db.postgresql_password}"
     LETTER_TRACKING_DB_NAME       = "${module.db.postgresql_database}"
+    FLYWAY_URL                    = "jdbc:postgresql://${module.db.host_name}:${module.db.postgresql_listen_port}/${module.db.postgresql_database}"
```

Comment:
I could build flyway properties based on the LETTER_TRACKING... ones, but I imagine that at some point they will become different, as the API shouldn't be allowed to change schema.